src/analysis/max_min_driving_force.jl

```diff
@@ -0,0 +1,163 @@
+"""
+    max_min_driving_force(
+        model::StandardModel,
+        thermodynamic_data::Dict{String, Float64},
+        optimizer;
+        modifications = [],
+        proton_id = "h_c",
+        water_id = "h2o_c",
+        concentration_ratios = [
+            ("atp_c", "adp_c", 10.0),
```

Comment:
these concentrations are somehow standardized around the world?

Reply:
yep, these are "physiological" ratios of these important cofactors often assumed in papers/measured in practice

Reply:
I'll add units for the concentrations though